web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/AutoDeploy.tsx

```diff
@@ -268,36 +266,22 @@ export function AutoDeploy({ toggleDeployMethod }: DeployServiceProp) {
 
             <StyledBox mb={5}>
               <header>
-                <H3>Step 4 (Optional)</H3>
-                <Subtitle3 mb={2}>Define Matcher Labels</Subtitle3>
-              </header>
-              <Labels
-                labels={labels}
-                setLabels={setLabels}
-                disableBtns={attempt.status === 'processing'}
-                showLabelMatchErr={showLabelMatchErr}
-                dbLabels={dbLabels}
-                autoFocus={false}
-                region={dbMeta.selectedAwsRdsDb?.region}
-              />
-            </StyledBox>
-
-            <StyledBox mb={5}>
-              <header>
-                <H3>Step 5</H3>
+                <H3>Step 4</H3>
                 <Subtitle3 mb={2}>
                   Deploy the Teleport Database Service
                 </Subtitle3>
               </header>
               <ButtonSecondary
-                width="215px"
+                width="230px"
                 type="submit"
                 onClick={() => handleDeploy(validator)}
-                disabled={attempt.status === 'processing'}
+                disabled={isProcessing}
                 mt={2}
                 mb={2}
               >
-                Deploy Teleport Service
+                {isDeploying
+                  ? 'Redeploy Teleport Service'
```

Comment:
When I'm in the story for this component and I click on "Redeploy Teleport Service", my position on the page is moved up. Is this supposed to happen? Maybe it has to do with some elements temporarily disappearing and then appearing again because `attempt.status` got updated?

https://github.com/user-attachments/assets/0ad12ab8-e444-4ff5-929d-b9aae4a564fd

Reply:
i had to screen record this and see where the jump was coming from, but i think it's because when user hits `redeploy` the hint box hides during processing, but it happens so fast in stories that you couldn't see it.

it takes some ms for this to succeed during production though.